common/app/common/package.scala

```diff
@@ -14,6 +14,10 @@ import play.twirl.api.Html
 
 object `package` extends implicits.Strings with implicits.Requests with play.api.mvc.Results {
 
+  def isCommercialExpiry(error: ErrorResponse): Boolean = {
```

Comment:
As I mentioned in my email, this is quite likely to blow up when using Thrift :(

CAPI will return the error response in JSON and the Scala client will try to parse it as Thrift.

Let's chat IRL next week about how to work around this.